client/my-sites/checkout/checkout-thank-you/marketplace/use-themes-thank-you-data.tsx

```diff
@@ -48,26 +60,23 @@ export function useThemesThankYouData(
 		};
 	}, [ dispatch, siteId ] );
 
-	const themesSection: ThankYouSectionProps = {
-		sectionKey: 'theme_information',
-		nextSteps: themesList
-			.filter( ( theme ) => theme )
-			.map( ( theme ) => ( {
-				stepKey: `theme_information_${ theme.id }`,
-				stepSection: (
-					<ThankYouThemeSection
-						theme={ theme }
-						isOnboardingFlow={ isOnboardingFlow }
-						continueWithPluginBundle={ continueWithPluginBundle }
-					/>
-				),
-			} ) ),
-	};
+	const themesSection: React.ReactElement[] = themesList
```

Comment:
```suggestion
	const themesSection = themesList
```
Typescript will infer the correct type here anyway.

Reply:
I'll also remove it for https://github.com/Automattic/wp-calypso/pull/86364/files#diff-0f74f28cf46b329608b6edf14805d41168036242b17839b85e988dafb821bb72R169